include/mbedtls/threading.h

```diff
@@ -24,6 +24,14 @@
 #ifndef MBEDTLS_THREADING_H
 #define MBEDTLS_THREADING_H
 
+/*
+ * Ensure gmtime_r is available even with -std=c99; must be defined before
+ * config.h, which pulls in glibc's features.h. Harmless on other platforms.
+ */
+#if !defined(_POSIX_C_SOURCE)
+#define _POSIX_C_SOURCE 200112L
```

Comment:
Potential problem: another header might have previously defined `_POSIX_C_SOURCE` to a smaller value, or a subsequent header might want to define `_POSIX_C_SOURCE` to a larger value.